cypress/support/fragments/marcAuthority/marcAuthority.js

```diff
@@ -113,6 +113,22 @@ export default {
         .fillIn(content),
     ]);
   },
+  checkFieldByRow: (rowIndex, field) => {
```

Comment:
Can you use QuickMarcEditor.verifyTagFieldAfterUnlinking() except creating new one.

Reply:
fixed